src/test/java/org/folio/test/support/AccountsActionAPITests.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.test.support;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.notNullValue;
+
+import io.restassured.response.Response;
+import java.util.Date;
+import org.folio.rest.jaxrs.model.Account;
+import org.folio.rest.utils.ResourceClient;
+
+public abstract class AccountsActionAPITests extends ApiTests {
```

Comment:
Maybe we can just call it `ActionsAPITests`. It would be consistent with our names for the "new" actions functionality - class `Action`, for example.

Reply:
It makes sense.  I will change the name